File: TestDatabaseFirst/Models/PostComments.cs
using System;
using System.Collections.Generic;

namespace TestDatabaseFirst.Models
{
    public partial class PostComments
    {
        public int PostCommentId { get; set; }
        public string Content { get; set; }
        public int PostId { get; set; }
        public int UserId { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }

        public Posts Post { get; set; }
        public Users User { get; set; }
    }
}

File: TestDatabaseFirst/ClientApp/src/app/_models/postcomments.ts
import { Posts } from "./posts";
import { User } from "./user";

export class PostComments {
  PostCommentId: Number;
  Content: String;
  PostId: Number;
  UserId: Number;
  CreatedAt?: Date;
  UpdatedAt?: Date;

  Post: Posts;
  User: User;
}
